src/commands/cmd_server.cc

```diff
@@ -823,16 +823,33 @@ class CommandScan : public CommandScanBase {
     if (args.size() % 2 != 0) {
       return {Status::RedisParseErr, errWrongNumOfArguments};
     }
```

Comment:
Please remove this.

Reply:
I am a C++beginner. Thank you very much for your guidance on my code. I have learned a lot